pkg/servicemesh/federation/server/routing.go

```diff
@@ -74,14 +74,15 @@ func (s *meshServer) removeServiceFromAuthorizationPolicy(target *federationmode
 	}
 	for index, host := range apSpec.Rules[0].To[0].Operation.NotHosts {
 		if host == target.Hostname {
-			apSpec.Rules[0].To[0].Operation.NotHosts = append(apSpec.Rules[0].To[0].Operation.NotHosts[:index], apSpec.Rules[0].To[0].Operation.NotHosts[index+1:]...)
+			apSpec.Rules[0].To[0].Operation.NotHosts = append(apSpec.Rules[0].To[0].Operation.NotHosts[:index],
+				apSpec.Rules[0].To[0].Operation.NotHosts[index+1:]...)
```

Comment:
Is there any risk of an OutOfRange expection here and in the range statement above? Better add guards?

Reply:
see above:
```
	if len(apSpec.Rules) != 1 || len(apSpec.Rules[0].To) != 1 {
		return fmt.Errorf("invalid AuthorizationPolicy for service export")
	}
```